app/views/tags/index.html.erb

```diff
@@ -1,3 +1,45 @@
 <% content_for :head do %>
   <%= raw crawlable_meta_data(title: @title, description: @description_meta) %>
 <% end %>
+
+<div class="tag-crawler">
+
+  <% if @extras[:categories] %>
+    <% @extras[:categories].each do |category| %>
+      <div class="tag-list">
+        <% if category[:name] %>
+          <h3><%= category[:name] %></h3>
+        <% end %>
+        <% category[:tags].each do |tag| %>
+          <%= render "tag", tag: tag %>
+        <% end %>
+      </div>
+      <div class="clearfix"></div>
+    <% end %>
+  <% end %>
+
+  <% if @extras[:tag_groups] %>
+    <% @extras[:tag_groups].each do |tag_group| %>
+      <div class="tag-list">
+        <% if tag_group[:name] %>
+          <h3><%= tag_group[:name] %></h3>
+        <% end %>
+        <% tag_group[:tags].each do |tag| %>
+          <%= render "tag", tag: tag %>
+        <% end %>
+      </div>
+      <div class="clearfix"></div>
+    <% end %>
+  <% end %>
+
+  <% if @tags && !@tags.empty? %>
```

Comment:
This can be shortened to

```suggestion
  <% if @tags.present? %>
```

Reply:
I have made the change @ZogStriP, sorry for the delay :bowing_man:

Reply:
@ZogStriP ping